llama_stack/cli/stack/_build.py

```diff
@@ -210,16 +210,17 @@ def run_stack_build_command(args: argparse.Namespace) -> None:
                 )
                 sys.exit(1)
 
-        if build_config.image_type == LlamaStackImageType.CONTAINER.value and not args.image_name:
-            cprint(
-                "Please specify --image-name when building a container from a config file",
-                color="red",
-            )
-            sys.exit(1)
+        if build_config.image_type == LlamaStackImageType.CONTAINER.value:
```

Comment:
This check seems redundant since even without it, down the line we get the following error:

```
$ USE_COPY_NOT_MOUNT=true LLAMA_STACK_DIR=. CONTAINER_BINARY=podman llama stack build --config tests/ci/custom-distro.yaml
Error building stack: Please specify an image name when building a container image without a template
```

So maybe just remove it.